be/src/olap/lru_cache.h

```diff
@@ -246,15 +252,19 @@ struct LRUHandle {
     struct LRUHandle* prev = nullptr;      // previous entry in lru list
     size_t charge;
     size_t key_length;
-    size_t total_size; // including key length
-    size_t bytes;      // Used by LRUCacheType::NUMBER, LRUCacheType::SIZE equal to total_size.
-    bool in_cache;     // Whether entry is in the cache.
+    size_t total_size; // Entry charge, used to limit cache capacity, LRUCacheType::SIZE including key length.
+    size_t bytes;  // Used by LRUCacheType::NUMBER, LRUCacheType::SIZE equal to total_size.
+    bool in_cache; // Whether entry is in the cache.
     uint32_t refs;
     uint32_t hash; // Hash of key(); used for fast sharding and comparisons
     CachePriority priority = CachePriority::NORMAL;
     MemTrackerLimiter* mem_tracker;
     LRUCacheType type;
+    // Save the last visit time of this cache entry.
+    // Use atomic because it may be modified by multi threads.
+    int64_t last_visit_time;
```

Comment:
atomic or not?

Reply:
`last_visit_time` only be modified in `LRUCache::insert` and `LRUCache::lookup`, in the cache lock.

Reply:
Then better to remove the comment.

Reply:
Good! I forgot to change the comment.

Reply:
done